style.css

```diff
@@ -0,0 +1,108 @@
+@font-face {
+  font-family: Poppins;
+  src: url("./resources/font/Poppins-Regular.ttf");
+}
+
+@font-face {
+  font-family: Poppins-Bold;
+  src: url("./resources/font/Poppins-Bold.ttf");
+}
+
+@media only screen and (max-width: 600px) {
+  body {
+    font-family: Poppins, sans-serif;
+    font-style: normal;
+    margin: 0;
+  }
+}
+
+html {
+  background: #e5e5e5;
+}
+
+.article {
+  background-image: url("./resources/images/header-shapes\ mobilebackground-Main.png");
+  display: flex;
+  flex-direction: column;
+  min-height: 100vh;
+  background-size: 100%;
+  background-repeat: no-repeat;
+  background-position: 1% 1%;
+  align-items: flex-start;
+  position: absolute;
+  border-bottom-left-radius: 15%;
+  background-color: #fff;
+}
```

Comment:
- [OPTIONAL] It would be nice to reposition the background image so that your design looks identical to that of the Figma design that you chose. 😎 💻 

 | Your Design | Figma Design |
| ------------- | ------------- |
| ![Screenshot_3](https://github.com/Uzair-Manzoor/Portfolio/assets/77106962/d687243e-903d-4167-a763-f5057812977f) | ![Screenshot_4](https://github.com/Uzair-Manzoor/Portfolio/assets/77106962/7cdf123c-69a4-4243-b544-92d71ca026cc) |